src/vs/workbench/contrib/files/browser/fileActions.contribution.ts

```diff
@@ -587,7 +588,7 @@ MenuRegistry.appendMenuItem(MenuId.MenubarFileMenu, {
 	group: '1_new',
 	command: {
 		id: NEW_UNTITLED_FILE_COMMAND_ID,
-		title: nls.localize({ key: 'miNewFile', comment: ['&& denotes a mnemonic'] }, "New &&Query")
+		title: locConstants.fileActionsContributionMiNewQuery
```

Comment:
there are a lot of places that do not have the {{SQL CARBON EDIT}} tag.

Reply:
The reason for this is that there's a comment above that already indicated the string was changed.